modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions in user-provisioned infrastructure
+
+When you assign the contributor and user access administrator roles to the service principal, you automatically grant all the required permissions. To deploy and destroy an {product-title} cluster with user-provisioned infrastructure, the service principal requires the following permissions.
```

Comment:
Similar highlight should be applied here also, for name of the roles. Thanks!

Reply:
Addressed